ros2_ws/config/InstallAicaDescriptions.cmake

```diff
@@ -0,0 +1,16 @@
+macro(install_aica_descriptions)
```

Comment:
Why not

```suggestion
macro(install_aica_descriptions BASE_FOLDER OUT_FOLDER)
```

and avoid dealing with `ARGX`?

Reply:
Good idea, didn't know it was possible